var/da/da_define_structures/da_random_seed.inc

```diff
@@ -77,7 +80,8 @@ subroutine da_random_seed
    else                                 ! Random seed set "randomly"
       call random_seed
       call random_seed(get=seed_array(1:seed_size))
-      write(unit=message(1),fmt='(a,10i16)') 'Random number seed array = ', seed_array
+      write(fmtstring, '(a,i3,a)') '(a27,', seed_size, '(i16))'
+      write(unit=message(1),fmt=trim(fmtstring)) 'Random number seed array = ', (seed_array(i), i=1, seed_size)
       call da_message(message(1:1))
```

Comment:
@davegill Dave, as Michael Duda did in setup_physics_suite in module_check_a_mundo.F, building a run-time format string allows writing formatted text for variable array size.

Reply:
@jamiebresch @mgduda 
Jamie,
Yep, I like it.